### PR TITLE
fix: make images media type instead of text

### PR DESCRIPTION
**Pull request by ryunsong-contentful**

### CODE REVIEW

File: packages/core/src/definitions/styles.ts [hunk 1]
```diff
@@ -452,7 +452,7 @@ export const singleColumnBuiltInStyles: Partial<
 };
 
 export const columnsBuiltInStyles: Partial<
-  Record<ContainerStyleVariableName, ComponentDefinitionVariable<'Text' | 'Boolean'>>
+  Record<ContainerStyleVariableName, ComponentDefinitionVariable<'Text' | 'Media' | 'Boolean'>>
```
Comment (ryunsong-contentful) [inline packages/core/src/definitions/styles.ts:455]: Should be reverted

File: packages/core/src/definitions/styles.ts [hunk 2]
```diff
@@ -313,7 +313,7 @@ export const containerBuiltInStyles: Partial<
 };
 
 export const singleColumnBuiltInStyles: Partial<
-  Record<ContainerStyleVariableName, ComponentDefinitionVariable<'Text' | 'Boolean'>>
+  Record<ContainerStyleVariableName, ComponentDefinitionVariable<'Text' | 'Media' | 'Boolean'>>
```
Comment (ryunsong-contentful) [inline packages/core/src/definitions/styles.ts:316]: All non image related definitions should not contain media

File: packages/core/src/definitions/styles.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { ComponentDefinitionVariable, ContainerStyleVariableName } from '@/types';
 
 export const builtInStyles: Partial<
-  Record<ContainerStyleVariableName, ComponentDefinitionVariable<'Text' | 'Boolean'>>
```
Comment (elylucas) [inline packages/core/src/definitions/styles.ts:4]: Good catch getting all these instances of background images. I didn't think of that.

**Review by elylucas — CHANGES_REQUESTED** [review]
I see the background images handled in here, but the `Image` component itself will also need its src url changed to media here: https://github.com/contentful/experience-builder/blob/development/packages/components/src/components/Image/index.ts#L25

**Pull request closed without merging**